content/recipes/introduction/FAIR-and-knowledge-graphs.md

```diff
@@ -249,22 +223,19 @@ alt: wikidata sparql endpoint
 A SPARQL query over Wikidata RDF endpoint: Cell lines with names that could also be URLs (Internet of Cell Lines)
 ```
 ````
-
-For more information about SPARQL queries over Wikidata, see [here](https://www.wikidata.org/wiki/Wikidata:SPARQL_query_service/queries/examples#Cell_lines_with_names_that_could_also_be_URLs_\(Internet_of_Cell_Lines\)).
-     
 ### 2. Property graphs
     
-Like `RDF graphs`, `Property graphs` are too used to build `Knowledge graphs`.
-However, unlike `RDF graphs`, `Property graphs` allows the vertices (aka the `edge`) to carry *annotations*,
+Like `RDF graphs`, `property graphs(PG)` are too used to build `knowledge graphs`.
+However, unlike `RDF graphs`, `property graphs` allows the vertices (aka the `edge`) to carry *annotations*,
 which can be queried. These `annotations` are called `properties` hence the denomination `Property graphs`,
 also referred to as **`Labeled Property Graphs`** or **`LPG`**.
```

Comment:
Seems confusing where you define property graph or labeled property graphs? Are they the same or different?

Reply:
I would suggest to chose one term and using it consistently throughout the recipe.

Reply:
I agree but felt it was necessary to highlight the variation in names, simply to provide that 'handle'/ guideline for newcomers.